src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,107 @@
+.header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding: 24px 370px;
+
+    position: relative;
+    width: 1920px;
+    height: 88px;
+    
+    /* Primaty/Grey Light */
+    background: #F6F5F4;
+  }
+  
+.content {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+
+    width: 1180px;
+    height: 40px;
```

Comment:
1. don't need to set `flex-direction: row;`, `width: 1180px;`, `height: 40px;`
2. add `min-width: 1180px;`

Reply:
https://github.com/taodemy/taotalk/blob/457af72abe43f54f6b3c079e56f580a15ec67f7f/src/components/Header/Header.module.scss#L13-L18

Previous codes are modified as suggestions in guidance. And other similar problems in css file are also modified accordingly.